### PR TITLE
Fixes to get tests passing

### PR DESCRIPTION
**Pull request by odow**
Replaces #82

### CODE REVIEW

File: test/MOI_wrapper.jl [hunk 1]
```diff
@@ -43,6 +43,7 @@ using CDDLib
                     # TODO Should be fixed in MOI master
                     "test_model_LowerBoundAlreadySet",
                     "test_model_UpperBoundAlreadySet",
+                    "test_linear_open_intervals",
```
Comment (blegat) [inline test/MOI_wrapper.jl:46]: Why is that needed for `Float64` ?

Comment (odow) [reply test/MOI_wrapper.jl]: I just had it in the wrong place. Removed because it should work with MOI v1.15
